### PR TITLE
Add maxWidth option

### PR DESCRIPTION
**Pull request by amishas157**
Though the actual width of scale is not in control of developer, but there is a possibility of keeping the maxWidth of scale as flexible.

### CODE REVIEW

File: js/ui/control/scale.js [hunk 1]
```diff
@@ -12,8 +12,10 @@ module.exports = Scale;
  * @class Scale
  * @param {Object} [options]
  * @param {string} [options.position='bottom-left'] A string indicating the control's position on the map. Options are `'top-right'`, `'top-left'`, `'bottom-right'`, and `'bottom-left'`.
+ * @param {integer} [options.maxWidth='150'] A number indicating the maximum length of the scale control in pixels.
```
Comment (mourner) [inline js/ui/control/scale.js]: `integer` -> `number`; you could also remove `A number indicating ` and preserve the meaning fully

File: js/ui/control/scale.js [hunk 2]
```diff
@@ -26,30 +28,35 @@ Scale.prototype = util.inherit(Control, {
...
+function updateScale(map, scale, options) {
     // A horizontal scale is imagined to be present at center of the map
-    // container with maximum length (Also the initial length) as 100px.
+    // container with maximum length (Default) as 100px.
     // Using spherical law of cosines approximation, the real distance is
     // found between the two coordinates.
+    var maxWidth = 100;
+    if (options && options.maxWidth) {
+        maxWidth = options.maxWidth;
+    }
```
Comment (mourner) [inline js/ui/control/scale.js]: You could write this shorter:

```js
var maxWidth = options && options.maxWidth || 100;
```

**Comment by amishas157** [comment]
Thanks @mourner for the review.

**Comment by lucaswoj** [comment]
Thanks @amishas157! ❤️

**Pull request merged**